src/app/pages/waiting-room/cat-adi-part2/waiting-room.cat-adi-part2.page.ts

```diff
@@ -33,6 +32,6 @@ export class WaitingRoomCatAdiPart2Page implements OnInit {
   }
 
   async navigateForward(): Promise<void> {
-    await this.routeByCat.navigateToPage(TestFlowPageNames.COMMUNICATION_PAGE, TestCategory.ADI2);
+    await this.routeByCat.navigateToPage(TestFlowPageNames.COMMUNICATION_PAGE);
```

Comment:
this routeByCategoryProvider is used everywhere across pr where typically would be router